volpiano_display_utilities/text_volpiano_alignment.py

```diff
@@ -6,238 +6,317 @@
-    chant_text: str,
-    volpiano: str,
-    text_presyllabified: bool = False,
-    clean_text: bool = False,
+    chant_text: str, volpiano: str, clean_text: bool, text_presyllabified: bool = False
 ) -> List[Tuple[str, str]]:
     """
     Aligns syllabified text with volpiano, performing periodic sanity checks
     and accounting for misalignments.
 
     chant_text [str]: the text of a chant
     volpiano [str]: the volpiano for a chant
-    text_presyllabified [bool]: whether the text is already syllabified. Passed
-        to syllabify_text (see that functions documentation for more details). Defaults
-        to False.
     clean_text [bool]: whether to clean the text before syllabifying. Passed to
         syllabify_text (see that functions documentation for more details). Defaults
         to False.
+    text_presyllabified [bool]: whether the text is already syllabified. Passed
+        to syllabify_text (see that functions documentation for more details). Defaults
+        to False.
 
     returns [list[tuple[str, str]]]: list of tuples of text syllables and volpiano syllables
         as (text_str, volpiano_str)
     """
-    syllabified_text: List[List[List[str]]] = syllabify_text(
-        chant_text,
-        clean_text=clean_text,
-        flatten_result=False,
-        text_presyllabified=text_presyllabified,
+    syllabified_text = syllabify_text(
+        chant_text, clean_text=clean_text, text_presyllabified=text_presyllabified
     )
-    # Performs some validation on the passed volpiano string
-    preprocessed_volpiano: str = _preprocess_volpiano(volpiano)
-    # Section volpiano to match text sections returned by syllabify_text
-    # Split at clefs, barlines, and missing music markers, removing empty
-    # sections created by the split.
-    volpiano_sections = VOLPIANO_SECTIONING_REGEX.split(preprocessed_volpiano)
-    volpiano_sections = list(filter(lambda x: x != "", volpiano_sections))
-    logging.debug("Volpiano sections: %s", volpiano_sections)
-    if len(volpiano_sections) == len(syllabified_text) + 2:
-        # Add the opening clef with no text
-        comb_text_and_vol = [("", volpiano_sections[0])]
-        # For each interior section, align the section
-        for vol_sec, txt_sec in zip(volpiano_sections[1:-1], syllabified_text):
-            aligned_section: List[Tuple[str, str]] = _align_section(txt_sec, vol_sec)
-            comb_text_and_vol.extend(aligned_section)
-        # Add the final barline with no text
-        comb_text_and_vol.append(("", volpiano_sections[-1]))
-    else:
-        raise ValueError(
-            """Volpiano and text sections do not match. Ensure appropriate
-            barlines and section markers are present in both."""
+    preprocessed_volpiano, fin_bar = preprocess_volpiano(volpiano)
+    syllabified_volpiano = syllabify_volpiano(preprocessed_volpiano)
+    # Add the opening clef with no text
+    alignment: List[Tuple[str, str]] = [("", "1---")]
```

Comment:
Could this variable be given a more descriptive name? `aligned_text_and_volpiano_syllables` (if I'm understanding correctly)?